Misc/NEWS.d/next/Security/2022-06-16-12-13-55.gh-issue-87389.MS9wAR.rst

```diff
@@ -0,0 +1,8 @@
+:mod:`http.server`: Fix an open redirection vulnerability in the HTTP server
+when an URI path starts with ``//``.  Vulnerability discovered, and initial
+fix proposed, by Hamza Avvan.  Change :func:`urllib.parse.urlunsplit` to
+sanitize ``path`` argument in order to avoid confusing the first component of
+the path as a net location or scheme.  Add :func:`urllib.parse.pathsplit`
+function.
+
+Co-authored-by: Gregory P. Smith <gps@google.com>
```

Comment:
I try not to publish this email address lest people think they can contact me there. "Gregory P. Smith [Google]" is what I attempt to type when I'm conscious of things.